The fields in JobConfigurationPOJO missing default values

## Bug Report

**For English only**, other languages will not accept.

Before report a bug, make sure you have:

- Searched open and closed [GitHub issues](https://github.com/apache/shardingsphere-elastic-job-lite/issues).
- Read documentation: [ElasticJob Doc](http://shardingsphere.apache.org/elasticjob/docs/elastic-job-lite/00-overview/).

Please pay attention on issues you submitted, because we maybe need more details. 
If no response anymore and we cannot reproduce it on current information, we will **close it**.

Please answer these questions before submitting your issue. Thanks!

### Which version of ElasticJob did you use?
Branch - master

### Which project did you use? ElasticJob-Lite or ElasticJob-Cloud?
ElasticJob-Lite

### Expected behavior

### Actual behavior
```
[ERROR] 2020-07-14 08:31:00,033 --simpleJob_Worker-1-- [org.apache.shardingsphere.elasticjob.infra.handler.error.impl.LogJobErrorHandler] Job 'simpleJob' exception occur in job processing 
org.apache.shardingsphere.elasticjob.infra.exception.JobExecutionEnvironmentException: Time different between job server and register center exceed '0' seconds, max time different is '0' seconds.
	at org.apache.shardingsphere.elasticjob.lite.internal.config.ConfigurationService.checkMaxTimeDiffSecondsTolerable(ConfigurationService.java:103) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJobFacade.checkJobExecutionEnvironment(LiteJobFacade.java:81) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.execute(ElasticJobExecutor.java:85) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.execute(LiteJob.java:52) [classes/:na]
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202) [quartz-2.3.2.jar:na]
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:na]
[ERROR] 2020-07-14 08:31:00,043 --scriptJob_Worker-1-- [org.apache.shardingsphere.elasticjob.infra.handler.error.impl.LogJobErrorHandler] Job 'scriptJob' exception occur in job processing 
org.apache.shardingsphere.elasticjob.infra.exception.JobExecutionEnvironmentException: Time different between job server and register center exceed '0' seconds, max time different is '0' seconds.
	at org.apache.shardingsphere.elasticjob.lite.internal.config.ConfigurationService.checkMaxTimeDiffSecondsTolerable(ConfigurationService.java:103) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJobFacade.checkJobExecutionEnvironment(LiteJobFacade.java:81) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.executor.ElasticJobExecutor.execute(ElasticJobExecutor.java:85) ~[classes/:na]
	at org.apache.shardingsphere.elasticjob.lite.internal.schedule.LiteJob.execute(LiteJob.java:52) [classes/:na]
	at org.quartz.core.JobRunShell.run(JobRunShell.java:202) [quartz-2.3.2.jar:na]
	at org.quartz.simpl.SimpleThreadPool$WorkerThread.run(SimpleThreadPool.java:573) [quartz-2.3.2.jar:na]

```

### Reason analyze (If you can)
The default value of `maxTimeDiffSeconds` is `-1` in `JobConfiguration.Builder`. If `JobConfigurationPojo` missing default value, it would be set to `0` and ERROR occurred.

### Steps to reproduce the behavior.
Using `JobConfigurationPOJO.toJobConfiguration()` to create a instance of `JobConfiguration`. Then create a instance of `ScheduleJobBootstrap` and start scheduling.

### Example codes for reproduce this issue (such as a github link).

--- a/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/config/pojo/JobConfigurationPOJO.java
+++ b/elasticjob-lite/elasticjob-lite-core/src/main/java/org/apache/shardingsphere/elasticjob/lite/internal/config/pojo/JobConfigurationPOJO.java
@@ -46,7 +46,7 @@ public final class JobConfigurationPOJO {
     
     private boolean misfire;
     
-    private int maxTimeDiffSeconds;
+    private int maxTimeDiffSeconds = -1;
     
     private int reconcileIntervalMinutes;
     

--- a/elasticjob-lite/elasticjob-lite-core/src/test/java/org/apache/shardingsphere/elasticjob/lite/internal/config/pojo/JobConfigurationPOJOTest.java
+++ b/elasticjob-lite/elasticjob-lite-core/src/test/java/org/apache/shardingsphere/elasticjob/lite/internal/config/pojo/JobConfigurationPOJOTest.java
@@ -38,7 +38,7 @@ public final class JobConfigurationPOJOTest {
             + "jobName: test_job\n"
             + "jobParameter: param\n"
             + "jobShardingStrategyType: AVG_ALLOCATION\n"
-            + "maxTimeDiffSeconds: 0\n"
+            + "maxTimeDiffSeconds: -1\n"
             + "misfire: false\n"
             + "monitorExecution: false\n"
             + "overwrite: false\n"
@@ -52,7 +52,7 @@ public final class JobConfigurationPOJOTest {
             + "disabled: false\n"
             + "failover: false\n"
             + "jobName: test_job\n"
-            + "maxTimeDiffSeconds: 0\n"
+            + "maxTimeDiffSeconds: -1\n"
             + "misfire: false\n"
             + "monitorExecution: false\n"
             + "overwrite: false\n"
@@ -130,6 +130,7 @@ public final class JobConfigurationPOJOTest {
         actual.setShardingTotalCount(3);
         actual.setShardingItemParameters("0=A,1=B,2=C");
         actual.setJobParameter("param");
+        actual.setMaxTimeDiffSeconds(-1);
         actual.setJobShardingStrategyType("AVG_ALLOCATION");
         actual.setJobExecutorServiceHandlerType("CPU");
         actual.setJobErrorHandlerType("IGNORE");
@@ -144,6 +145,7 @@ public final class JobConfigurationPOJOTest {
         actual.setJobName("test_job");
         actual.setCron("0/1 * * * * ?");
         actual.setShardingTotalCount(3);
+        actual.setMaxTimeDiffSeconds(-1);
         assertThat(YamlEngine.marshal(actual), is(YAML_WITH_NULL));
     }
     

